vcd/resource_vcd_org.go

```diff
@@ -610,9 +622,9 @@ func resourceVcdOrgImport(ctx context.Context, d *schema.ResourceData, meta inte
 		return nil, fmt.Errorf(errorRetrievingOrg, err)
 	}
 
-	diagErr := setOrgData(d, vcdClient, adminOrg)
-	if diagErr != nil {
-		return []*schema.ResourceData{}, fmt.Errorf("error setting Org data: %v", diagErr)
+	diags := setOrgData(d, vcdClient, adminOrg)
+	if diags != nil && diags.HasError() {
+		return []*schema.ResourceData{}, fmt.Errorf("error setting Org data: %v", diags)
```

Comment:
I guess we don't care for ignore, but just to not that warnings are lost. OTOH, import requires `error` only so that is fine

Reply:
Yes, that is unfortunate, that Import does not support Diagnostics.